conf/messages.cy

```diff
@@ -1112,8 +1112,6 @@ paymentHistory.paymentToHmrc                                    = Taliad roeddec
 paymentHistory.cutOver                                          = Credyd o flwyddyn dreth gynharach
 paymentHistory.mfa                                              = Credyd o ganlyniad i addasiad gan CThEF
 paymentHistory.repaymentInterest                                = Credyd o log ar ad-daliadau
-paymentHistory.POA1RR-credit                                    = Taliad cyntaf ar gyfrif: credyd o’ch Ffurflen Dreth
```

Comment:
why we drop these two lang resources ? they been added this morning

Reply:
This is the result of a pull from main. Currently checking if this is correct